src/test/health-check/index.ts

```diff
@@ -43,8 +43,18 @@ const cases: Case[] = [
         check: ['title', 'content', 'timestamp.created'],
         related: ['#77'],
     },
-    // TODO: 나우뉴스
-    // TODO: 네이버뉴스
+    {
+        impl: '나우뉴스',
+        url: 'https://nownews.seoul.co.kr/news/newsView.php?id=20211117601017',
+        check: ['title', 'content', 'timestamp.created', 'timestamp.lastModified'],
+        related: [],
+    },
+    {
+        impl: '네이버뉴스',
+        url: 'https://news.naver.com/main/read.naver?mode=LSD&mid=shm&sid1=101&oid=018&aid=0005090507',
+        check: ['title', 'content', 'timestamp.created', 'timestamp.lastModified'],
```

Comment:
`check` 란에는 해당 기사의 내용에 보이는 요소를 넣어주면 됩니다.
이 기사에는 마지막으로 수정된 날짜에 대한 정보가 들어있지 않은 것 같습니다.
기사 내용에 기자 이름과 이메일이 보입니다. 하지만 기자 이메일은 기계적으로 쉽게 알아낼 수 있는지가 확실치 않아서 기자의 이름에 대해서 까지만 테스트가 이뤄지면 좋을 것 같습니다.
```suggestion
        check: ['title', 'content', 'timestamp.created', 'reporters.0.name'],
```

Reply:
위 댓글은 수정 제안인데, `Commit suggestion ▼` 버튼을 누르고 `Commit changes` 버튼을 이어서 눌러주시면 수정 제안이 이 PR에 바로 적용됩니다.